### ISSUE TITLE
Existing tag is not updated with value from NSI

### ISSUE THREAD

**Issue by j13m126** [Zverik/every_door#561]
I reactivated a [poi (change #5)](https://www.openstreetmap.org/node/2327282792/history) by tapping on the header bar and searching for its name (it's listed in the [nsi](https://nsi.guide/index.html?t=brands&k=amenity&v=bicycle_rental&tt=wienmobil#wienmobilrad-cda769)) but later noticed that the network tag didn't get updated, while network:wikidata got set, causing an inconsistency.

**Comment by mnalis** [comment]
I can confirm. It seems EveryDoor will not overwrite existing data with data from NSI when selecting a preset.

e.g. If I create a node with
```
amenity=bicycle_rental
network=old net
```

and save, and then try to change it to NSI-provided [Stadtrad Innsbruck](https://nsi.guide/index.html?t=brands&k=amenity&v=bicycle_rental&tt=stadtrad%20i#stadtradinnsbruck-cda769)  which contains `amenity=bicycle_rental`, `brand`, `network` and `operator`, the new version would be tagged:

```
amenity=bicycle_rental
brand=Stadtrad Inssbruck
network=old net
operator=Nextbike
```


i.e. it would **add** missing tags (`brand`, `operator`) from NSI, but it would not modify **existing** ones (like `network`).

**Comment by gdabski** [comment]
I reported this before in #493.

**Comment by Zverik** [comment]
Fixed in 3.1. Only values that were changed by hand won't get changed when applying a different preset.
